core/src/edu/cornell/gdiac/main/Animator.java

```diff
@@ -0,0 +1,62 @@
+package edu.cornell.gdiac.main;
+
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.g2d.TextureRegion;
+import edu.cornell.gdiac.assets.AssetDirectory;
+import edu.cornell.gdiac.physics.obstacle.SimpleObstacle;
+import edu.cornell.gdiac.util.FilmStrip;
+
+public class Animator {
+  public static FilmStrip frames;
+  public static PlayerModel player;
+  public static FanModel fan;
```

Comment:
also why are you holding references to the `PlayerModel` and the `FanModel` as public static variables? This is unsafe since they can be set to null anywhere in the program. As I said I am just a bit confused by this class overall, much more documentation is needed and please reconsider some of your design decisions (sorry if this sounds harsh I just don't really understand what you're doing).